src/main/java/uk/gov/hmcts/juror/api/moj/service/report/JurySummoningMonitorReportServiceImpl.java

```diff
@@ -50,21 +52,38 @@ public JurySummoningMonitorReportResponse viewJurySummoningMonitorReport(
 
                 if (result != null && !result.isEmpty()) {
                     List<String> values = List.of(result.split(","));
-                    setupByPoolFields(response, values);
+                    setupResponse(response, values);
                 }
 
             } catch (Exception e) {
                 log.error("Error getting jury summoning monitor report by pool", e);
                 throw new MojException.InternalServerError("Error getting jury summoning monitor report by pool", e);
             }
         } else {
-            // TODO implement the logic to get the report data by court once the function is available
 
+            String courtLocCodes = jurySummoningMonitorReportRequest.isAllCourts()
+                ?
+                String.join(",", courtQueriesRepository.getAllCourtLocCodes())
```

Comment:
Do we need to exclude the bureau court here?

Reply:
There wont be anyone summoned to that location code, but have filtered out "400" in the query